.github/workflows/aap_chatbot_build_and_deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: AAP Chatbot Stack - Build and Deploy
```

Comment:
As a nit-pick; I wonder if "Build and *Deploy*" is correct?

You yourself state this PR is for "...creating the base llama stack image, and **publishing** into quay..."

IDK if "Build and release" or "Build and publish" is better?

I know we use "deploy" in other repositories meaning "publish" so at lease this is consistent :smiley: